parsec/core/mountpoint/fuse_runner.py

```diff
@@ -167,6 +167,7 @@ def _run_fuse_thread():
                             str(mountpoint_path.absolute()),
                             foreground=True,
                             encoding=encoding,
+                            default_permissions=True,
```

Comment:
Whether it is `default_permissions=True` or `defer_permissions=True` for macos only, we need a detailed comment explaining our investigation. In particular, we should mention the role of `gid` in the original issue, so we don't forget about it.